index.html

```diff
@@ -893,9 +893,12 @@
 This is a special case of the more general principle that data should not be used for more
 [=purposes=] than the data's subjects understood it was being collected for.
 
-A service should explain how it uses people's data to protect them and other people, and
-how it might additionally use someone's data if it believes that person has broken the
-rules.
+Services sometimes use people's data in order to protect those or other people.
+This is legitimate as long as the extra risk created by the data collection is
+less than the protection it offers and the extra risk isn't concentrated on
+[=vulnerable people=]. A service that does this should explain what data it's
```

Comment:
In order to merge some improvement quickly, we have a suggestion to:
```suggestion
A service that does this should explain what data it's
```

Then I'll send another PR about the legitimacy of the practice.